### ISSUE TITLE
question about quantize model

### ISSUE THREAD

**Issue by yeliang2258** [microsoft/onnxruntime#11961]
**Describe the bug**
A clear and concise description of what the bug is. To avoid repetition please make sure this is not one of the known issues mentioned on the respective release page.

Why can't the following Gems be merged into QGemm? 
![image](https://user-images.githubusercontent.com/30516196/175230775-39912aa9-612a-4901-ac6e-90f5b0680620.png)

I read GemmNodeGroupSelector class, it should meet the merge condition.
![image](https://user-images.githubusercontent.com/30516196/175231208-b8b76774-b996-4c65-b110-1a8ba8926ce4.png)

**Urgency**
If there are particular important use cases blocked by this or strict project-related timelines, please share more information and dates. If there are no hard deadlines, please specify none.

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Ubuntu16.04
- ONNX Runtime installed from (source or binary): binary
- ONNX Runtime version: 1.11.1
- Python version: python3.7

**Comment by yeliang2258** [comment]
Does the bias of Gemm also need dequantize?

**Comment by yufenglee** [comment]
@yeliang2258, yes, currently merging logic requires bias to be quantized. How do you generate the quantized model? With tf2onnx?

**Comment by yeliang2258** [comment]
This model is from Paddle2ONNX.
